packages/odyssey-react/src/components/Box/Box.theme.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  // Font
+  FontFamilyBase: theme.FontFamilyBase,
+  FontWeightNormal: theme.FontWeightNormal,
+  FontStyleNormal: theme.FontStyleNormal,
+  FontSizeBase: theme.FontSizeBase,
+  FontLineHeightBase: theme.FontLineHeightBase,
+  ColorTextBody: theme.ColorTextBody,
+
+  // Space
+  SpaceRemXs: theme.SpaceRemXs,
+  SpaceRemS: theme.SpaceRemS,
+  SpaceRemM: theme.SpaceRemM,
+  SpaceRemL: theme.SpaceRemL,
+  SpaceRemXl: theme.SpaceRemXl,
+
+  // Border
+  BorderWidthBase: theme.BorderWidthBase,
+  BorderStyleBase: theme.BorderStyleBase,
+  BorderRadiusBase: theme.BorderRadiusBase,
+  BorderRadiusOuter: theme.BorderRadiusOuter,
+  ColorBorderBase: "transparent",
+  ColorBorderUi: theme.ColorBorderUi,
+  ColorBorderDisplay: theme.ColorBorderDisplay,
+  ColorBorderPrimary: theme.ColorBorderPrimary,
+  ColorBorderDanger: theme.ColorBorderDanger,
+
+  // Focus Outline
+  ColorFocusPrimary: theme.ColorFocusPrimary,
+  ColorFocusDanger: theme.ColorFocusDanger,
+  FocusWidth: "0.125rem",
```

Comment:
Some good examples in this theme module of "component only" tokens, these are values that can be overriden but are not derived from any backing global theme token. Whether these *should* have global backing values is entirely up for debate of course but this is one strategy to help us move away from compile time only sass variables.

Reply:
I like this setup as a holding pen. IMO we may want to move a few tokens into here until Design finishes auditing all of our variables. Box covers a ton of properties, but I'm not sure the components share a standard yet.

Reply:
@edburyenegren-okta I wish we had more time for a design audit but I don't believe we do. For now, let's move forward with what we have here with the understanding that this is all fluid and we can make changes once design has formulated an opinion.

Reply:
Is there a strong reason not to keep these Box-only tokens as Box-level values until we have the global standard figured out? (As `FocusWidth` here). Same question re: the background colors above. The reason there's a mismatch is because Box added the sass variables to give us hypothetical coverage - not because those colors are used everywhere. (Note their absence from `colors.scss` and `tokens.scss`.)

No reason to hold up Box or withTheme work for a design audit. We can promote them to the global tokens as we go.

My concern is that if we make them global, we are building our SOT around a single component. Implementers utilizing the general tokens will not have the context that these are fluid/changing/undecided.